bin/configs/aspnet-fastendpoints-petstore-new.yaml

```diff
@@ -0,0 +1,14 @@
+generatorName: aspnet-fastendpoints
+outputDir: samples/server/petstore/aspnet/fastendpoints
```

Comment:
what about adding the new folder to github workflow (e.g. https://github.com/OpenAPITools/openapi-generator/blob/master/.github/workflows/samples-dotnet8-server.yaml) so that CI will test it moving forward?

Reply:
Done.
I also added configuration file for each option available in the generator